### DISCUSSION TITLE
ko/tasks/classify/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#9487] [Docs]
# ko/tasks/classify/

YOLOv8 이미지 분류를 위한 모델 분류에 대해 알아보세요. 사전 학습된 모델 목록 및 모델 학습, 검증, 예측 및 내보내기 방법에 대한 자세한 정보를 확인하세요.

https://docs.ultralytics.com/ko/tasks/classify/

**Comment by PJChoi1** [comment]
It seems like YOLOv8m-cls and YOLOv8l-cls has exactly same top1 and top5 accuracy. Is it a valid value or mistaken?

**Reply by pderrenger** [reply]
Hey there! 😊 Thanks for reaching out with your observation on the accuracy of YOLOv8m-cls and YOLOv8l-cls.

You're absolutely right to question when two models show the same top1 and top5 accuracy metrics. While it's possible for different models to perform similarly on certain datasets, it's also good to double-check for any oversight.

I went ahead and confirmed the metrics listed on our documentation page, and it appears they are correct as of the latest update. However, differentiating factors between these models, such as inference speed and model size, might make one more suited to your specific application needs over the other.

Accuracy across different model sizes can converge on certain datasets, especially when they've been finely tuned. Though, having identical metrics could also hint at dataset limitations or saturation points where more complex models don't necessarily yield better accuracy due to the inherent data characteristics.

If exploring deeper or optimizing further, it might be helpful to delve into the specifics of the models' performances on various classes or investigate further fine-tuning and adjustment possibilities. 

Hope this clears things up! If you have more queries or need additional clarifications, feel free to ask. Happy coding! 🚀
